File: WPRK-Android/src/main/java/com/muse/wprk/main/model/Podcast.kt
package com.muse.wprk.main.model

import kotlinx.serialization.Serializable

@Serializable
data class Podcast(
    val id: String,
    val title: String,
    val description: String,
    val thumbnailURL: String,
    val episodesAvailable: Int
    )

File: WPRK-iOS/WPRK-iOS/Domain/Main/Model/Podcast.swift
//
//  Podcast.swift
//  WPRK-iOS
//
//  Created by Mwai Banda on 6/25/22.
//

import Foundation

struct Podcast: Identifiable, Hashable {
    var id: String
    var title: String
    var category: String
    var description: String
    var imageURL: String
    var episodesAvailable: Int
}
